rotkehlchen/db/dbhandler.py

```diff
@@ -3457,3 +3461,25 @@ def create_db_backup(self) -> Path:
             new_db_path,
         )
         return new_db_path
+
+    def add_connected_location(self, location: Location) -> None:
+        if location == Location.EXTERNAL:
+            # Ignore external location that can appear while importing from
+            # cointracking csv files
+            return
+        cursor = self.conn.cursor()
+        cursor.execute(
+            'INSERT OR IGNORE INTO connected_location (location) VALUES (?)',
+            (location.value,),
+        )
+        self.conn.commit()
+        self.update_last_write()
+
+    def get_connected_locations(self) -> List[Location]:
```

Comment:
This should change to a query of `DISTINCT` locations in trades, asset movements, ledger actions ... and any thing else? Perhaps ammswaps?